src/main/java/de/doubleslash/keeptime/view/ViewController.java

```diff
@@ -150,6 +150,7 @@ private class Delta {
    private final BooleanProperty mouseHoveringProperty = new SimpleBooleanProperty(false);
    public static final LongProperty activeWorkSecondsProperty = new SimpleLongProperty(0);
    public static final ObjectProperty<Color> fontColorProperty = new SimpleObjectProperty<>();
+   boolean contexMenuOpenBoolean = false;

```

Comment:
can you add appropriate modifiers to the property?

Reply:
Changed Modifiers to private static boolean contextMenuOpenBoolean